nmdc_runtime/site/translation/submission_portal_translator.py

```diff
@@ -61,9 +121,10 @@ def _get_pi(
             name=study_form.get("piName"),
             email=study_form.get("piEmail"),
             orcid=study_form.get("piOrcid"),
+            profile_image_url=self.study_pi_image_url,
         )
 
-    def _get_doi(self, metadata_submission: JSON_OBJECT) -> Union[List[str], None]:
+    def _get_doi(self, metadata_submission: JSON_OBJECT) -> Union[List[nmdc.Doi], None]:
```

Comment:
Another note to myself (@eecavanna):
- `nmdc.Doi` is a `class` defined in the `nmdc_schema` package (installed via PyPI)